src/callbacks_stage/positivity_adaptive_filter_dg1d.jl

```diff
@@ -0,0 +1,171 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+"""
+
+Adaptive filter for DGMulti to ensure the well-posedness of the entropy
+projection step.
+"""
+# TODO: only support 1D GaussSBP. Multidimension and general DGMulti in future PRs.
+function adaptive_filter!(u, filter,
+                          mesh::DGMultiMesh{1},
+                          equations::Union{AbstractCompressibleEulerEquations,CompressibleNavierStokesDiffusion2D},
```

Comment:
I can put a separate PR on 1D CNS if you would like. I have all the routines for 1D.

Reply:
Wouldn't a PR on CNS just involve the elixirs? I think we can add them here.

Reply:
Oh I mean seems like Trixi only support compressible_navier_stokes_2d.jl from a quick grep? If there is no 1D CNS equation I can help to add it in.

Reply:
Ah! That is a good point. Can you file an issue for this?

Reply:
Done.